### DISCUSSION TITLE
Show different version of site based on selected link

### DISCUSSION THREAD

**Discussion by tjperry07** [vercel/next.js#16267] [Help] [answered]
I'd like to use Next to build a documentation site. I need to add a drop-down for versioning. Is there a way to show a different set of pages based on the drop-down or version the user picks?

**Comment by jamesmosier** [comment, marked as answer]
Unfortunately there is no functionality like that built into Next.js, you'd have to build that custom. I have not seen any sites use Next for docs, other than Next's docs 😄   but they do not do any versioning.

**Reply by tjperry07** [reply]
Thanks for the insight.
